edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerModule.cs

```diff
@@ -41,6 +44,12 @@ public class DockerModule : IModule<DockerConfig>
         [JsonProperty(PropertyName = "restartPolicy")]
         public virtual RestartPolicy RestartPolicy { get; }
 
+        [JsonProperty(
+            PropertyName = "pullPolicy",
+            DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        [DefaultValue(Core.Constants.DefaultPullPolicy)]
```

Comment:
It isn't really needed if the default is 0 but I thought I'll add this so that we can control the default value using the DefaultPullPolicy constant in case we ever decide to.
Also, we also had a constant for RestartPolicy so I thought I'll follow the same pattern. What do you think?

Reply:
Not needed anymore, I'll remove it.